commands/get/resources/getter_test.go

```diff
@@ -0,0 +1,186 @@
+package resources
+
+import (
+	"errors"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/ginkgo/extensions/table"
+	resmocks "github.com/opencontrol/compliance-masonry/commands/get/resources/mocks"
+	"github.com/opencontrol/compliance-masonry/lib/common"
+	"github.com/opencontrol/compliance-masonry/lib/common/mocks"
+	"github.com/opencontrol/compliance-masonry/lib/opencontrol"
+	parserMocks "github.com/opencontrol/compliance-masonry/lib/opencontrol/mocks"
+	"github.com/opencontrol/compliance-masonry/tools/constants"
+	"github.com/opencontrol/compliance-masonry/tools/fs"
+	fsmocks "github.com/opencontrol/compliance-masonry/tools/fs/mocks"
+	"github.com/opencontrol/compliance-masonry/tools/mapset"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
+)
```

Comment:
This many imports seems like a red flag that the setup for these structs/functions are too complicated, though I don't have specific suggestions about how to simplify 😕